code/modules/food_and_drinks/recipes/tablecraft/recipes_lizard.dm

```diff
@@ -0,0 +1,389 @@
+/datum/crafting_recipe/food/tiziran_sausage
+	name = "Raw Tiziran blood sausage"
+	reqs = list(
+		/obj/item/food/meat/rawcutlet = 1,
+		/obj/item/food/meat/rawbacon = 1,
+		/datum/reagent/blood = 5,
+		/datum/reagent/consumable/salt = 2
+	)
+	result = /obj/item/food/raw_tiziran_sausage
+	subcategory = CAT_LIZARD
```

Comment:
Has the code that means only lizard chefs can cook this already been added? Or is this common pool recipe.

Reply:
These are all common pool since attempting to make it lizard only nearly drove me mad last night. If any miracle solutions present themselves I'd be happy to make it lizard only (with an accompanying cookbook for non-lizards).